app/src/main/AndroidManifest.xml

```diff
@@ -41,7 +41,7 @@
                 <action android:name="com.google.firebase.MESSAGING_EVENT" />
             </intent-filter>
         </service>
-        [END firebase_service]
+        <!--[END firebase_service]-->
```

Comment:
why did we commented out this line?

Reply:
because xml file was giving error with this.